### ISSUE TITLE
a extensao nao vai mais funcionar alguem sabe outra forma de assistir a crunchyroll gratuitamente?

### ISSUE THREAD

**Issue by Pedro1Developer** [Mateus7G/crp-iframe-player#102]
a extensao nao vai mais funcionar alguem sabe outra forma de assistir a crunchyroll gratuitamente?

**Comment by alvszlucy** [comment]
também queria saber

**Comment by alvszlucy** [comment]
mas por enquanto vou de animesonline.cc

**Comment by railanbarbosaaquino** [comment]
> mas por enquanto vou de animesonline.cc

eu recomendo mais o https://animesonlinex.co/ por causa da rapidez nos posts, se por o Energized Basic Protection no Ublock assiste de boas sem ad nenhum

**Comment by BLDarK** [comment]
Recomendo tbm o animestc.net, site muito boa e sem propagando usando o operaGX, assisti online e baixa tbm, eps sempre em 1080p.

**Comment by jandsonatan** [comment]
Eu disse em uma publicação abaixo que consegui assistir pelo site: vrv.co 
Ele é um site da crunchyroll também, para conseguir assistir eu utilizei uma extensão AdBlock e outra extensão de VPN dos EUA para navegar no site, já que ele não funciona no Brasil. Ao executar o episódio que eu queria, desligava a VPN para não influenciar na velocidade carregamento.

**Comment by BLDarK** [comment]
até os episódios premium? pq desse jeito ate pela crunchyroll da pra fazer só não rola os premium.
testei nesse vrv.co n conseguir ver os premium não.

**Comment by jandsonatan** [comment]
Verdade, percebi que somente alguns premium na Crunchyroll não são premium no VRV

**Comment by Matheus-Luan** [comment]
tem o animeszone.net tbm, é o meu preferido, é bem simples de acessar

**Comment by kvothepsx** [comment]
assista pelo animesvision, o melhor de todos, da pra baixar em full hd de graça tbm

**Comment by 43D** [comment]
Usava anime.vision, mas volta e meia eles cagam com site, to usando puray.moe quando se trata de animes da CR, lá eles tão usando os links da CR ainda

**Comment by kvothepsx** [comment]
> Usava anime.vision, mas volta e meia eles cagam com site, to usando puray.moe quando se trata de animes da CR, lá eles tão usando os links da CR ainda

vou testar aqui valeu

**Comment by kvothepsx** [comment]
> Usava anime.vision, mas volta e meia eles cagam com site, to usando puray.moe quando se trata de animes da CR, lá eles tão usando os links da CR ainda

legal mas não pra baixar igual no animes vision, uso la faz anos e nunca tive problemas, da pra baixar animes ate em 4k agora

**Comment by Matheus-Luan** [comment]
!!!!
Galera, olha ai: https://github.com/Mateus7G/crp-iframe-player/issues/109
Acho q por isso q parou a extensão... ent n tem mais volta dai

**Comment by 2c0f1** [comment]
> também queria saber

Da pra assistir no site da vcr.co usando VPN ou proxy no EUA e Canadá por oq eu testei, eu estou tentando implementar o uso de proxy na extensão e troca de vídeos na crunchyroll pela do vcr.co, mas eu não tenho muita experiência, talvez leve um tempo, mas se quiser assistir é só abrir o site que eu disse e usar uma vpn ou proxy
